views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+                        <td>id for group</td>
+                      </tr>
+                      <tr>
+                        <th>area</th>
+                        <td>name of area represented <code>Example County</code></td>
+                      </tr>
+                      <tr>
+                        <th>area_id<span class="superscript">&dagger;</span></th>
+                        <td>id for area</td>
+                      </tr>
+                      <tr>
+                        <th>term<span class="superscript">&dagger;</span></th>
+                        <td>id or index for term</td>
+                      </tr>
+                      <tr>
+                        <th>start_date</th>
+                        <td>start of membership (if needed) <code>2010-11-29</code></td>
+                      </tr>
+                      <tr>
+                        <th>end_date</th>
+                        <td>end of membership (if needed) <code>2012-07-14</code></td>
+                      </tr>
+                      <tr>
+                        <th>image</th>
+                        <td>URL to image file <code>http://example.com/example.jpg</code></td>
+                      </tr>
+                      <tr>
+                        <th>gender</th>
+                        <td>gender if known <code>M</code> or <code>F, or <code>male</code> or <code>female</code></td>
+                      </tr>
+                    </table>
+                  </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p>
+          <span class="superscript">&dagger;</span>
+          You can use fields marked &dagger; as IDs, although we can’t
+          guarantee they’ll always be unique <em>outside</em> this legislature.
+          For example, all politicians in this legislature with the same
+          <code>area_id</code> are representing the same area (which is what
+          you’d expect). Where appropriate, the actual value might be a useful
+          value. For example, we like to use <a
+          href="http://opencivicdata.readthedocs.org/en/latest/ocdids.html">Open
+           Civic Data IDs</a> like this:
+          <code>ocd-division/country:us/state:ca/cd:47</code>, but if no such
+          mapping is available from our sources, we’ll probably use a slug like
+          <code>area-foo-county</code>. If you can suggest better values whenever
+          you find us using our own ones, please get in touch!
+        </p>
+        <h2>
+          Legislatures and terms
+        </h2>
+        <p>
+          For every country, we break the data down into
+          <strong>legislatures</strong> (for example, the “parliament” or
+          “congress” of the country). Currently, we’re only working with
+          national, that is, top-level, legislatures (as opposed to local or
+          state-level legislatures). In the future we might include those too.
```

Comment:
slight scoping confusion here. The 'those' refers to the thing in the brackets from the previous sentence, which tripped me up slightly.